docs/dockerImages.md

```diff
@@ -15,6 +15,11 @@ Some questions may require additional customization of existing images, such as
 
 1. Install [Docker Desktop](https://docs.docker.com/desktop/) in your environment.
 2. Create a [Docker Hub](https://hub.docker.com/) account, if you don't yet have one.
+
+   !!! warning
+
+   This step should be done by a user that will maintain a long-term association with the course in question, such as a permanent faculty member or support staff, not a teaching assistant or sessional instructor. This will simplify any future updates to custom images, if they become necessary. If multiple users are expected to get access to push these images, you may consider the use of [access tokens](https://docs.docker.com/security/access-tokens/) or [teams](https://docs.docker.com/admin/organization/manage-a-team/).
```

Comment:
This is not causing a warning box because it is indented by 3 spaces, and the admonitions plugin [seems to require 4 spaces](https://github.com/squidfunk/mkdocs-material/discussions/6290#discussioncomment-7474767). Changing it to 4 spaces causes prettier to revert to 3. Working on a workaround.